INSTALL.md

```diff
@@ -3,94 +3,101 @@
+$ export VERSION=1.11.4 OS=linux ARCH=amd64  # change this as you need
+
+$ wget -O /tmp/go${VERSION}.${OS}-${ARCH}.tar.gz https://dl.google.com/go/go${VERSION}.${OS}-${ARCH}.tar.gz && \
+  sudo tar -C /usr/local -xzf /tmp/go${VERSION}.${OS}-${ARCH}.tar.gz
 ```
 
-Then extract the archive to `/usr/local` (or use other instructions on go
-installation page).
+Finally, set up your environment for Go:
 
 ```
-$ sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
+$ echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
+  echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
+  source ~/.bashrc
 ```
 
-Finally, set up your environment for go
+## Clone the repo
+golang is a bit finicky about where things are placed. Here is the correct way
+to build Singularity from source:
 
 ```
-$ echo 'export GOPATH=${HOME}/go' >> ~/.bashrc
-$ echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc
-$ source ~/.bashrc
+$ mkdir -p ${GOPATH}/src/github.com/sylabs && \
+  cd ${GOPATH}/src/github.com/sylabs && \
+  git clone https://github.com/sylabs/singularity.git && \
+  cd singularity
 ```
 
-## Clone the repo
-golang is a bit finicky about where things are placed. Here is the correct way
-to build Singularity from source.
+To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
 
 ```
-$ mkdir -p $GOPATH/src/github.com/sylabs
-$ cd $GOPATH/src/github.com/sylabs
-$ git clone https://github.com/sylabs/singularity.git
-$ cd singularity
+$ git checkout v3.0.2
 ```
 
 ## Compile the Singularity binary
 Now you are ready to build Singularity. Dependencies will be automatically
 downloaded. You can build Singularity using the following commands:
 
 ```
-$ cd $GOPATH/src/github.com/sylabs/singularity
-$ ./mconfig
-$ cd ./builddir
-$ make
-$ sudo make install
+$ cd ${GOPATH}/src/github.com/sylabs/singularity && \
+  ./mconfig && \
+  cd ./builddir && \
+  make && \
+  sudo make install
 ```
 
-Alternatively, to build an rpm on CentOS/RHEL use the following commands:
+And Thats it! Now you can check you Singularity version by running:
 
 ```
-$ sudo yum install -y rpm-build wget
-$ cd $GOPATH/src/github.com/sylabs/singularity
-$ ./mconfig
-$ make -C builddir rpm
+$ singularity version
 ```
 
-Golang doesn't have to be installed to build an rpm because the rpm
-build installs golang and all dependencies, but it is still recommended
-for a complete development environment.
+<br>
 
-To build a stable version of Singularity, check out a [release tag](https://github.com/sylabs/singularity/tags) before compiling:
+Alternatively, to build an RPM on CentOS/RHEL use the following commands:
 
 ```
-$ git checkout v3.0.2
+$ sudo yum install -y rpm-build wget
+
+$ cd ${GOPATH}/src/github.com/sylabs/singularity && \
+  ./mconfig && \
+  make -C builddir rpm
 ```
 
+Golang doesn't have to be installed to build an rpm because the rpm
```

Comment:
Change to "...build an RMP because the RPM..."

Capitalize 'rpm' to match previous instance (line 87).